src/scss/components/_footer.scss

```diff
@@ -0,0 +1,216 @@
+                margin-right: $margin-right--16;
+                margin-bottom: 0;
+
+                li {
+                    padding-left: 0;
+
+                    a {
+                        color: $color-secondary;
+                        text-decoration: none;
+                        font-weight: $font-weight-medium;
+                        line-height: $line-height--1-4;
+                        display: block;
+                    }
+                }
+            }
+        }
+
+        .locale-select-container {
+            padding: 0;
+            order: 1;
+            flex-shrink: 0;
+            margin-left: auto;
+
+            .form-group {
+                max-width: $max-width-600;
+            }
+        }
+
+        .form-label {
+            display: flex;
+            align-items: center;
+            margin: 0;
+            padding-bottom: 8px;
+            min-height: $min-height--28;
+        }
+
+        .sr-only {
+            position: absolute !important;
+            height: $height--1;
+            width: $width--1;
+            overflow: hidden;
+            clip: $form-label-clip;
+        }
+
+        .select-container {
+            position: relative;
+            min-width: $min-width--240;
+            max-width: $max-width-600;
+
+            .select-icon-container {
+                pointer-events: none;
+                display: flex;
+                align-items: center;
+                position: absolute;
+                top: 0;
+                height: $height-full;
+                padding: 0 8px;
+
+
+                &.select-icon-left {
+                    left: 0;
+                    justify-content: flex-end;
+                    width: $width--36;
+                }
+
+                &.select-icon-right {
+                    right: 0;
+                    width: $width--36;
+                }
+            }
+        }
+
+        .select {
+            background: $color-white;
+            border-radius: $border-radius--4;
+            border: $border-size-default solid $color-border-dark;
+            cursor: pointer;
+            display: block;
+            padding: 0 40px 0 12px;
+            width: $width-full;
+            appearance: none;
+            -moz-appearance: none;
+            -webkit-appearance: none;
+            height: $height--40;
+            padding-left: 36px;
+        }
+
+        .logo-and-copyright {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            padding: 64px 0 48px;
+
+            .logo-container {
+                display: flex;
+                align-items: center;
+                padding: 0;
+
+                img {
+                    max-width: $max-width-110;
+                }
+            }
+
+            .copyright-container {
+                padding: 0;
+                height: $height-full;
+                top: 50%;
+                color: #73726c;
```

Comment:
Do we have this color in `colors.scss`?
If we don't have, please make it useful.